packages/web_pointer_interceptor/README.md

```diff
@@ -0,0 +1,29 @@
+# web_pointer_interceptor
+
+A small Flutter widget to prevent clicks (in web) from being intercepted by underlying HtmlElement views.
+
+## How to use
+
+Wrap your button element in a `PointerInterceptor` widget, like so:
```

Comment:
This starts out a bit too specific. I'd start with more general use-cases (e.g. "use it if your app renders widgets on top of platform views that handle pointer events"), then start this section with "For example". I would also focus on scenarios rather than raw usage examples, something that answers the question _why_ one would use it. For example, it is not clear why one would want to wrap a button. However, if you first describe the scenario the example will make intuitive sense, e.g. start with "Let's say you'd like to render custom play/pause buttons on top of a video player. If you just draw on top of the `<video>` element both your buttons and the video player handle mouse clicks.", then talk about wrapping the button elements. The other use-case we should specifically call out is `<iframe>` and how it swallows pointer events.